### DISCUSSION TITLE
Change UID of user ncdata belongs to

### DISCUSSION THREAD

**Discussion by Joly0** [nextcloud/all-in-one#4677] [Ideas] [answered]
Hey, i know it isnt optimal, but i would like to have nextcloud accessible via smb. I have an user on the host that i use for smb access, with a specific uid.
Is it possible to configure nextcloud, so files belong to a user with a specific uid?

Linuxserver nextcloud image has the option to configure puid and pgid and that worked, but for ease of use i want to use nextcloud-aio and an option similar to that of the lsio image would be great.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/1974
